pkgs/development/python-modules/pulsar-client/default.nix

```diff
@@ -93,12 +93,12 @@ let
       "3.12-aarch64-darwin" = getSrcFromPypi {
         platform = "macosx_10_15_universal2";
         dist = "cp312";
-        hash = "sha256-9/jw/wr1oUD9pOadVAaMRL081iVMUXwVgnUMcG1UNvE=";
+        hash = "sha256-Hgd6SDm+Pq094/BbTCRCadyi3wf0fOoLkFRMfp3BZC8=";
```

Comment:
How come we have to update this despite the version not changing?

Reply:
¯\_(ツ)_/¯, that's a great question. 

I saw the hash mismatch when running `nixpkgs-review`, so I updated it.

Reply:
@fabaff Do you know?

Reply:
FWIW, the hash I updated to, is the same has the hash [reported](https://pypi.org/project/pulsar-client/3.4.0/#copy-hash-modal-a0783d68-9339-4e46-b025-340783370db3) on PiPy (after converting from base64 to hex encoding).

Reply:
That hash was never correct. 

I can reproduce it on the aarch64-darwin community builder.

```bash
error: build of '/nix/store/2sxwrknkxw34z2p3s9a7007rjzwnvafh-pulsar_client-3.4.0-cp312-cp312-macosx_10_15_universal2.whl.drv' on 'ssh://fab@darwin-build-box.nix-community.org' failed: hash mismatch in fixed-output derivation '/nix/store/2sxwrknkxw34z2p3s9a7007rjzwnvafh-pulsar_client-3.4.0-cp312-cp312-macosx_10_15_universal2.whl.drv':
         specified: sha256-9/jw/wr1oUD9pOadVAaMRL081iVMUXwVgnUMcG1UNvE=
            got:    sha256-Hgd6SDm+Pq094/BbTCRCadyi3wf0fOoLkFRMfp3BZC8=
```

Hmmm, we have `pulsar-client` and `pulsar`. Looks like that renaming `pulsar` to `pulsar-client` would help to get rid of the wheel stuff.

Reply:
Yeah, looks that way since it never build on Hydra either: https://hydra.nixos.org/build/255103381

So this shouldn't be a blocker on merging this PR I don't think.